homeassistant/components/wallbox/__init__.py

```diff
@@ -75,7 +76,7 @@ def _get_data(self):
         except requests.exceptions.HTTPError as wallbox_connection_error:
             raise ConnectionError from wallbox_connection_error
 
-    def _set_charging_current(self, charging_current):
+    def _set_charging_current(self, charging_current: str | float) -> None:
         """Set maximum charging current for Wallbox."""
```

Comment:
Why is this typed as `str | float`, I think this should always be numeric. Also, the number component passes a `float` as argument.

Reply:
I added it originally as `wallbox.setMaxCharingCurrent` string formats the value and thus the method would work regardless if called with a `str` or `float`. You're right though that it makes more sense as just `float`. Pushed a new commit for it.